webapp/components/wpt-amend-metadata.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright 2020 The WPT Dashboard Project. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+
+import '../node_modules/@polymer/paper-dialog/paper-dialog.js';
+import '../node_modules/@polymer/paper-input/paper-input.js';
+import '../node_modules/@polymer/paper-toast/paper-toast.js';
+import { html, PolymerElement } from '../node_modules/@polymer/polymer/polymer-element.js';
+import { LoadingState } from './loading-state.js';
+import { ProductInfo } from './product-info.js';
+
+class AmendMetadata extends LoadingState(ProductInfo(PolymerElement)) {
+  static get is() {
+    return 'wpt-amend-metadata';
+  }
+
+  static get template() {
+    return html`
+      <style>
+        img.browser {
+          height: 26px;
+          width: 26px;
+          position: relative;
+        }
+        paper-button {
+          text-transform: none;
+          margin-top: 5px;
+        }
+        paper-input {
+          text-transform: none;
+          align-items: center;
+          margin-bottom: 20px;
+        }
+        .metadataEntry {
+          display: flex;
+          align-items: center;
+        }
+        .link {
+          align-items: center;
+          color: white;
+        }
+      </style>
+      <paper-dialog id="dialog">
+        <h3>Triage Failing Test</h3>
+        <div class="metadataEntry">
+          <img class="browser" src="[[displayLogo(product)]]">
+          &nbsp; >> [[test]] : &nbsp;
+          <paper-input label="Bug URL" value="{{url}}" autofocus></paper-input>
+        </div>
+        <div class="buttons">
+          <paper-button onclick="[[close]]">Dismiss</paper-button>
+          <paper-button onclick="[[triage]]" dialog-confirm>Triage</paper-button>
+        </div>
+      </paper-dialog>
+      <paper-toast id="show-pr" duration="10000"><span id="msg"></span><a id="prLink" class="link" target="_blank"]"></a></paper-toast>
```

Comment:
`]`? Seems like a typo here?`

This is really a static template and we can directly put data binding in it instead of using the DOM API to overwrite its content.

```suggestion
      <paper-toast id="show-pr" duration="10000"><span>[[errorMessage]]</span><a id="pr-link" class="link" target="_blank" href="[[prLink]]">[[prText]]</a></paper-toast>
```

And then you can have three new properties `errorMessage`, `prLink` & `prText`. Data binding is one of the top reasons people use Polymer instead of rolling their own custom elements. This makes updating logic and data flow super clear.